sdk/core/core-amqp/src/util/checkNetworkConnection.ts

```diff
@@ -10,25 +10,25 @@ import { logger } from "../log";
  * @internal
  */
 export function checkNetworkConnection(host: string): Promise<boolean> {
-  return new Promise((res) => {
+  return new Promise((resolve) => {
     logger.verbose("Calling dns.resolve to determine network connection status.");
-    resolve(host, function(err: any): void {
+    dnsResolve(host, function(err: any): void {
       if (err) {
         logger.verbose(
           "Error thrown from dns.resolve in network connection check: '%s', %O",
           err.code || err.name,
           err
         );
         // List of possible DNS error codes: https://nodejs.org/dist/latest-v12.x/docs/api/dns.html#dns_error_codes
-        // Only when dns.resolve returns an error we expect to see when the network is down, resolve as 'false'.
+        // We only resolve with `false` when dnsResolve fails with a network disconnection.
```

Comment:
This seemed less confusing to me! Let me know how it looks.

Reply:
cc @chradek

Reply:
I like the direction the comment is going in, minor suggestion for more accuracy:
```suggestion
        // We only resolve with `false` when dnsResolve fails with an error we expect to see when the network is down.
```